doc/setup.md

```diff
@@ -205,15 +207,6 @@ parameters:
         victoire.io: fr
 ```
 
-Update the `parameters.yml` with correct values.
```

Comment:
The name of the file is wrong, we talk about `config.yml`here. Can you please rename it instead of removing all this config step ?